director/origin_api.go

```diff
@@ -42,7 +42,8 @@ import (
 type (
 	OriginAdvertise struct {
 		Name       string        `json:"name"`
-		URL        string        `json:"url"`
+		URL        string        `json:"url"`     // This is the url for origin's XRootD service and file transfer
+		WebURL     string        `json:"web_url"` // This is the url for origin's web engine and APIs
```

Comment:
What happens when the `WebURL` isn't set?  I think we should have a test for when this isn't set (as this will be the case when importing from topology) and ensure that the `web_url` isn't set.

Reply:
I thought `OriginAdvertise` struct is only used for pelican origin to register at director and the topology is not involved? I don't quite understand when `WebURL` will be empty in this struct. I do believe that `ServerAd` struct can have empty `WebURL` as that's where we set topology response to.